src/startupscriptgenerator/python/scriptgenerator.go

```diff
@@ -39,15 +38,16 @@ func (gen *PythonStartupScriptGenerator) GenerateEntrypointScript() string {
 	scriptBuilder.WriteString("cd " + gen.SourcePath + "\n\n")
 
 	common.SetEnvironmentVariableInScript(&scriptBuilder, "PORT", gen.BindPort, DefaultBindPort)
-	packageSetupBlock := gen.getPackageSetupCommand()
+	buildManifest := common.GetBuildManifest(gen.SourcePath)
```

Comment:
@gpcastro I refactored this based on our offline discussion. Locally all integration tests passed, so I think my changes are good. I also kicked off a Nightly build now.